ibmdbpy/statistics.py

```diff
@@ -582,15 +582,12 @@ def describe(idadf):
 
         """
         table_name= idadf.internal_state.current_state
-        args_string = "('intable="+table_name+" , outtable="+table_name+"_temp');"
-        create_summary_query = "CALL nza..SUMMARY1000"+args_string
-        corr_df = idadf.ida_query(create_summary_query)
-        result_query = "SELECT * FROM "+table_name+"_temp ORDER BY columnname; "
-
-        corr_df = idadf.ida_query(result_query)
-        drop_result_query = "CALL nza..DROP_SUMMARY1000('intable="+table_name+"_temp');"
-        value = idadf.ida_query(drop_result_query)
-        return corr_df
+        outtable_name = idadf._idadb._get_valid_tablename(prefix="pyida_describe")
```

Comment:
The new describe function still returns a different format than the pandas counterpart and the values for the quantiles are missing. But the output of both functions should be the same. If not, we should give this function a different name.

For example, for the iris data:
Describe for idaDataframe:
```
>>> idadf = IdaDataFrame(idadb, 'IRIS') 
>>> idadf.describe()

    COLUMNNAME  DISTINCTVALUES  ... NONMISSINGCASES  MISSING
0        CLASS             3.0  ...             150        0
1           ID             NaN  ...             150        0
2  PETALLENGTH             NaN  ...             150        0
3   PETALWIDTH             NaN  ...             150        0
4  SEPALLENGTH             NaN  ...             150        0
5   SEPALWIDTH             NaN  ...             150        0

[6 rows x 13 columns]
```
Describe for pandas dataframe:
```
>>> df = idadf.as_dataframe()
>>> df.describe()
               ID  SEPALLENGTH  SEPALWIDTH  PETALLENGTH  PETALWIDTH
count  150.000000   150.000000  150.000000   150.000000  150.000000
mean    75.500000     5.843333    3.054000     3.758667    1.198667
std     43.445368     0.828066    0.433594     1.764420    0.763161
min      1.000000     4.300000    2.000000     1.000000    0.100000
25%     38.250000     5.100000    2.800000     1.600000    0.300000
50%     75.500000     5.800000    3.000000     4.350000    1.300000
75%    112.750000     6.400000    3.300000     5.100000    1.800000
max    150.000000     7.900000    4.400000     6.900000    2.500000
```